src/test/java/VolodymyrHrytsiuk/Tests/CrowellExperiencePractise.java

```diff
@@ -0,0 +1,47 @@
+package VolodymyrHrytsiuk.Tests;
+
+import BasePageAndTestInit.TestInit;
+import VolodymyrHrytsiuk.Tests.Pages.HomePageCrowell;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CrowellExperiencePractise extends TestInit {
+
+    @Test
+    public void experiencePractiseAdvertisingAndMediaCheck() {
+        HomePageCrowell homePageCrw = new HomePageCrowell(driver);
+        Actions action = new Actions(driver);
+
+        openUrl("https://www.crowell.com/");
+        homePageCrw.experienceButton().click();
+        homePageCrw.advertisingAndMediaSelect().click();
+        action.doubleClick(homePageCrw.contactsDeploy()).perform();
+        scroll(600);
+        homePageCrw.falseAdvertDisAndLitig().click();
+        action.doubleClick(homePageCrw.contactsDeploy()).perform();
+        driver.navigate().back();
```

Comment:
Please move all driver.navigate().back() to TestInit. Create a custom method something like:
_Public void navigateBack(){driver.navigate().back();}_ and use it instead
same for double click